guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,8 +12,19 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+
+.Prerequisites for Disconnected Environments
+* If you run {Project} in a disconnected environment, ensure it meets the following prerequisites:
+** You must obtain and deploy Leapp metadata manually.
+ For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+** You require access to {RHEL} and {Project} packages.
+** Obtain the ISO files for {RHEL} 8 and {Project}.
+For more information, see xref:upgrading_a_disconnected_satellite[].
+** For more information on customizing the Leapp upgrade for your environment, see https://access.redhat.com/articles/4977891[Customizing your {RHEL} in-place upgrade].
```

Comment:
```suggestion
If you run {Project} in a disconnected environment, ensure it meets the following prerequisites:
* You must obtain and deploy Leapp metadata manually.
 For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
* You require access to {RHEL} and {Project} packages.
* Obtain the ISO files for {RHEL} 8 and {Project}.
For more information, see xref:upgrading_a_disconnected_satellite[].
* For more information on customizing the Leapp upgrade for your environment, see https://access.redhat.com/articles/4977891[Customizing your {RHEL} in-place upgrade].
```
Personally, I wouldn't indent the bulleted list here. Let me know what you think about this.

Reply:
I like it better your way. Making the change.